examples/shard-worker.config.example

```diff
@@ -5,9 +5,16 @@ port: 8981
 # by all shard cluster servers
 public_name: "localhost:8981"
 
-# This determines whether the worker is a shard of the cas.
-# When not a shard, it will upload into the Sharded CAS
-omit_from_cas: false
+
+capabilities {
```

Comment:
I get the sensibility of omission flags, but given that we're rewriting this, let's make these positive assertions: 'cas', 'execution'

Reply:
fixed